gramps/gen/db/dummydb.py

```diff
@@ -1286,151 +1286,170 @@ def iter_citations(self):
         """
         if not self.db_is_open:
             LOG.debug("database is closed")
-        return []
+        return
+        yield
```

Comment:
I wouldn't do that. The `yield` is unreachable.

What about just doing `yield from ()` (without `return`)? Same below

Reply:
mypy is happy with `yield from ()`.
I'll update the PR accordingly. Thanks.